metadata-ingestion/src/datahub/ingestion/api/decorators.py

```diff
@@ -93,10 +93,20 @@ def capability(
     """
 
     def wrapper(cls: Type) -> Type:
-        if not hasattr(cls, "__capabilities"):
+        if not hasattr(cls, "__capabilities") or any(
```

Comment:
Can we just use single underscore instead of double underscore here? I think that's why we have to do this in the first place: https://www.scaler.com/topics/double-underscore-python/

Reply:
in this case, we actually do want double underscore to signal that it's specific to that class and not subclasses, since the inheritance piece of it we do ourselves using copies to avoid mutating the same underlying object

also the double underscores don't actually get mangled here because we're not within a class context